test/values.test.js

```diff
@@ -255,6 +255,42 @@ describe('Values', () => {
     )
   })
 
+  test('pretty print Match', () => {
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx')))),
+      'Query(Lambda("_", Match(Index("idx"))))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str'))),
+      'Query(Lambda("_", Match(Index("idx"), "str")))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str', 10))),
+      'Query(Lambda("_", Match(Index("idx"), ["str", 10])))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), ['str', 10]))),
+      'Query(Lambda("_", Match(Index("idx"), ["str", 10])))'
```

Comment:
looking the result, looks good to me, but what you guys think? use brackets always or remove always when rendering the expression? both are equivalente

Reply:
I have no strong opinions on that... I would say to use `[]` always to be more explicit? I'll ask on #developer-experience channel to see what other folks think about this.

Reply:
@marrony looks like we are going to use `[]` is that ok for you we merge this?